benchmark/perf_hooks/histogram.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common.js');
+
+const { createHistogram } = require('perf_hooks');
+
+const bench = common.createBenchmark(main, {
+  n: [1e5],
+  operation: ['creation', 'clone'],
+});
+
+let _histogram;
+
+function main({ n, operation }) {
+  switch (operation) {
+    case 'creation': {
+      bench.start();
+      for (let i = 0; i < n; i++)
+        _histogram = createHistogram();
+      bench.end(n);
+
+      // Avoid V8 deadcode (elimination)
+      assert.ok(_histogram);
+      break;
+    }
+    case 'clone': {
+      const histogram = createHistogram();
+
+      bench.start();
+      for (let i = 0; i < n; i++)
+        _histogram = structuredClone(histogram);
```

Comment:
Just for curiosity. Why this benchmark is relevant? It seems just `creation` would make sense here.

Reply:
No reason at all, I can split this benchmark in two (`creation` and `clone`).

If this is a blocker for you, I can add a fixup.

But I will open another PR to optimize `clone`, in that time I can split the benchmarks.

Reply:
Nevermind, I updated the benchmarks, and I forgot that I already optimize the clone.